### PR TITLE
User login

### LINKED ISSUE #2: User Login
The login page is displayed based on the following triggers:

1. The user taps on the login box.
2. The user initiates an action which necessitates them logging in first.

On the User is redirected to login screen, the login form should appear.
The users will login by providing the following details on the login form:

1. Email address
2. Password

Once they are successfully logged in, they should be redirected to the page that they had previously accessed. If they hadn't visited any page before then, they should be redirected to the crowdfunding page as a default.

The login form should also have the forgotten password and create account links on it.

### PR DESCRIPTION
**Pull request by Ugerpaul17**

### CODE REVIEW

File: lib/helpers/style.dart [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+Color light = const Color(0xFFF7F8FC);
+Color lightGrey = const Color(0xFFA4A6B3);
+Color dark = const Color(0xFF363740);
+Color active = const Color(0xFF3C19C0);
+Color lightBlue = const Color(0xFF01579B);
```
Comment (ajoluvya) [inline lib/helpers/style.dart]: I would love to see this color match with the blue on the logo.

**Review by ajoluvya — CHANGES_REQUESTED** [review]
I would also love to see the logo have rounded corners, can we apply styles to that effect?

File: lib/helpers/style.dart (hunk 1 above)
Comment (Ugerpaul17) [reply lib/helpers/style.dart]: the changes you requested has been fixed

**Review by ajoluvya — COMMENTED** [review]
This looks fine.

File: lib/helpers/style.dart (hunk 1 above)
Comment (ajoluvya) [reply lib/helpers/style.dart]: Yes, I notice the change. However, based on the image shared on WhatsApp, can we now have the colors for the button and the text changed to the orange? Ultimately, we should be having an additional color of orange to the palette.

Then you should also be adding those buttons for social logins as in the shared image.
Also consider putting the same exact wordings as in the shared image.

**Review by ajoluvya — COMMENTED** [review]
I'm awaiting the screenshots before I can get this closed.
Otherwise we seem to be on track.

**Review by ajoluvya — APPROVED** [review]
We'll move on for now, as we still have to implement the server end point calls.
Let's move on to the registration form.

**Pull request merged**